lib/logger.rb

```diff
@@ -156,8 +156,8 @@ def tty?
   end
 end
 
-def warn(str)
-  $stdout.warn str
+def warn(str, trace = nil)
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)